### PR TITLE
[CHI-126] Add activity component

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

File: src/website/views/components/activity.pug [hunk 1]
```diff
@@ -0,0 +1,1105 @@
...
+            .m-activity__end
+              .m-activity__end-header
+                strong trent.anderson
+                |  deployed
+                strong  Active Directory
+              .m-activity__end-content
+                ul.-stories
+                  li
+                    div
+                      .a-icon.-text--primary
```
Comment (mattnickles) [inline src/website/views/components/activity.pug]: There's 16 occurrences of `-text--primary` used on checkmark icons in the feed examples. Can you change them all to `-text--success` to match the design?

File: src/website/views/components/activity.pug [hunk 2]
```diff
@@ -0,0 +1,1105 @@
...
+    .m-activity.-feed
+      .m-activity__day
+        .m-activity__content
+          .m-activity__item
+            .m-activity__item-avatar
+              .a-avatar
+                img(src='../../assets/images/avatar.jpg', alt="avatar")
+            .m-activity__end
+              .m-activity__end-header
+                strong trent.anderson
```
Comment (mattnickles) [inline src/website/views/components/activity.pug]: The name (e.g. trent.anderson) and the artifact (e.g. Active Directory) could be plain text or a link. We need to ensure that whether it's a `strong` or an `a` tag, it renders with a `font-weight: 600;` (semi-bold).

File: src/chi/components/activity/activity.scss [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      }
+    }
+  }
+
+  &.-feed {
+    & .m-activity__item {
+      padding: 1.25rem 0;
+
+      .m-activity__start {
+        min-width: unset;
```
Comment (jllr) [inline src/chi/components/activity/activity.scss]: @davidojedacl unset property will not work in IE11. Please change it.

File: src/chi/components/activity/activity.scss [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+    .m-activity__content {
+      .m-activity {
+        &__item {
+          align-items: start;
+          border-bottom: $border solid set-color(grey, 20);
+          flex-direction: row;
+          display: flex;
+          justify-content: start;
+          padding: 1.25rem 0;
```
Comment (jllr) [inline src/chi/components/activity/activity.scss]: @davidojedacl padding top and bottom must be 1.5rem

File: src/chi/components/activity/activity.scss [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+      & .a-avatar {
+        margin-right: 1rem;
+      }
+    }
+
+    &.-compact {
+      & .m-activity__item {
+        align-items: center;
+        padding: 0.5rem 0;
```
Comment (jllr) [inline src/chi/components/activity/activity.scss]: @davidojedacl padding-top and bottom must be 0.75rem

File: src/chi/components/activity/activity.scss [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+            border-bottom: 0;
+          }
+        }
+      }
+    }
+  }
+
+  &.-feed {
+    & .m-activity__item {
+      padding: 1.25rem 0;
```
Comment (jllr) [inline src/chi/components/activity/activity.scss]: @davidojedacl this padding is duplicated, please remove it.

File: src/chi/components/activity/activity.scss [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+            & div {
+              display: flex;
+              align-items: center;
+
+              & .a-icon {
+                color: set-color(grey, 50);
+                margin-right: 1.5rem;
+              }
+
+              & .a-avatar {
```
Comment (jllr) [inline src/chi/components/activity/activity.scss]: @davidojedacl this class is not needed, please remove it. It is being inherited from l.26

File: src/chi/components/activity/activity.scss [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+        border-bottom: 0.0625rem solid rgba(set-color(black), 0.12);
+
+        &:last-child {
+          border-bottom: 0;
+        }
+      }
+      & .m-activity__item {
+        padding: 1.5rem 0;
+
+        & .a-avatar {
```
Comment (jllr) [inline src/chi/components/activity/activity.scss]: @davidojedacl this class is not needed, as it is being inherited from l.129

File: src/website/views/components/activity.pug [hunk 9]
```diff
@@ -0,0 +1,1105 @@
...
+              div
+                .a-avatar
+                  img(src='../../assets/images/avatar.jpg', alt="avatar")
+            .m-activity__end
+              .m-activity__end-header
+                strong trent.anderson
+                |  deployed
+                strong  Active Directory
+              .m-activity__end-content
+                ul.-stories
```
Comment (jllr) [inline src/website/views/components/activity.pug]: @davidojedacl is this needed for any purpose?  If don't, please remove it.

File: src/website/views/components/activity.pug [hunk 10]
```diff
@@ -0,0 +1,1105 @@
...
```
Comment (jllr) [inline src/website/views/components/activity.pug]: @davidojedacl is this needed for any purpose?  If don't, please remove it.

File: src/website/views/components/activity.pug [hunk 11]
```diff
@@ -0,0 +1,1105 @@
...
+                <div class="a-avatar">
+                  <img src="../../assets/images/avatar.jpg" alt="avatar">
+                </div>
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-content">
+                <ul class="-stories">
```
Comment (jllr) [inline src/website/views/components/activity.pug]: @davidojedacl is this needed for any purpose?  If don't, please remove it.

File: src/website/views/components/activity.pug [hunk 12]
```diff
@@ -0,0 +1,1105 @@
...
```
Comment (jllr) [inline src/website/views/components/activity.pug]: @davidojedacl is this needed for any purpose?  If don't, please remove it.

File: src/website/views/components/activity.pug [hunk 13]
```diff
@@ -0,0 +1,1105 @@
...
+                    use(xlink:href='#icon-circle-play-outline')
+                .a-avatar
+                  img(src='../../assets/images/avatar.jpg', alt="avatar")
+            .m-activity__end
+              .m-activity__end-header
+                strong trent.anderson
+                |  deployed
+                strong  Active Directory
+              .m-activity__end-content
+                ul.-stories
```
Comment (jllr) [inline src/website/views/components/activity.pug]: @davidojedacl is this needed for any purpose?  If don't, please remove it.

File: src/website/views/components/activity.pug [hunk 14]
```diff
@@ -0,0 +1,1105 @@
...
```
Comment (jllr) [inline src/website/views/components/activity.pug]: @davidojedacl is this needed for any purpose?  If don't, please remove it.

File: src/website/views/components/activity.pug [hunk 15]
```diff
@@ -0,0 +1,1105 @@
...
+                <div class="a-avatar">
+                  <img src="../../assets/images/avatar.jpg" alt="avatar">
+                </div>
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-content">
+                <ul class="-stories">
```
Comment (jllr) [inline src/website/views/components/activity.pug]: @davidojedacl is this needed for any purpose?  If don't, please remove it.

File: src/website/views/components/activity.pug [hunk 16]
```diff
@@ -0,0 +1,1105 @@
...
+                  </svg>
+                </div>
+                <div class="a-avatar"><img src="../../assets/images/avatar.jpg" alt="avatar"></div>
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-content">
+                <ul class="-stories">
```
Comment (jllr) [inline src/website/views/components/activity.pug]: @davidojedacl is this needed for any purpose?  If don't, please remove it.

File: src/chi/components/activity/activity.scss (hunk 3 above)
Comment (davidojedacl) [reply src/chi/components/activity/activity.scss]: done!

File: src/chi/components/activity/activity.scss (hunk 4 above)
Comment (davidojedacl) [reply src/chi/components/activity/activity.scss]: done!

File: src/chi/components/activity/activity.scss (hunk 5 above)
Comment (davidojedacl) [reply src/chi/components/activity/activity.scss]: done!

File: src/chi/components/activity/activity.scss (hunk 6 above)
Comment (davidojedacl) [reply src/chi/components/activity/activity.scss]: done!

File: src/website/views/components/activity.pug (hunk 9 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: I think is clearer to separate header footer and content but I change it without problem

File: src/website/views/components/activity.pug (hunk 16 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

File: src/website/views/components/activity.pug (hunk 15 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

File: src/website/views/components/activity.pug (hunk 14 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

File: src/website/views/components/activity.pug (hunk 13 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

File: src/website/views/components/activity.pug (hunk 12 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

File: src/website/views/components/activity.pug (hunk 11 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

File: src/website/views/components/activity.pug (hunk 10 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

File: src/chi/components/activity/activity.scss (hunk 7 above)
Comment (davidojedacl) [reply src/chi/components/activity/activity.scss]: done

File: src/chi/components/activity/activity.scss (hunk 8 above)
Comment (davidojedacl) [reply src/chi/components/activity/activity.scss]: done

File: src/website/views/components/activity.pug (hunk 9 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

File: src/website/views/components/activity.pug (hunk 10 above)
Comment (davidojedacl) [reply src/website/views/components/activity.pug]: done

**Pull request merged**